examples/blog-apollo-server/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "blog-apollo-server",
+  "version": "0.1.0",
+  "dependencies": {
+    "apollo-server": "^2.9.16",
+    "graphql-jit": "^0.4.2",
```

Comment:
point to the GraphQL-jit from `../../` (or) remove this dependency and replace imports to use `../../`

Reply:
Done. I had to upgrade `graphql` to `^14.5.3` for this code to compile though. Is this ok with you?

Reply:
The upgrade is okay, but I wonder why it was required?

Reply:
Main culprit in this case is `GraphQLSchema`. Graphql-jit was expecting older model and apparently there are some differences that compiler detected. Downgrading `apollo-server` should also help if you need more testing with latest `graphql-js`.